src/main/java/edu/kit/ipd/dbis/gui/filter/FilterUI.java

```diff
@@ -19,7 +19,7 @@
 
 public class FilterUI extends JPanel {
 	private final FilterController filterController;
-	private FilterManagement filterManagement;
+	private UIFilterManager UIFilterManager;
```

Comment:
why is the name of that variable capitalized (and the same as its type)?

Reply:
Because of IntelliJ's automatic refactoring; I'll quickly change that